### PR TITLE
Bug fix

### PR DESCRIPTION
**Pull request by naoyam**

### CODE REVIEW

File: torch/csrc/jit/codegen/cuda/ir_nodes.cpp [hunk 1]
```diff
@@ -1223,7 +1223,12 @@ class ConcretizeDomain : private BackwardVisitor {
 };
 
 void ConcretizeDomain::concretizePwOp(Expr* e) {
-  TensorView* tv = *ir_utils::filterByType<TensorView>(e->outputs()).begin();
+  if (e->output(0)->getValType() != ValType::TensorView) {
```
Comment (shmsong) [inline torch/csrc/jit/codegen/cuda/ir_nodes.cpp:1226]: I would prefer using ir_utils::isTVOp, which includes all related checks.

File: torch/csrc/jit/codegen/cuda/ir_nodes.cpp [hunk 2]
```diff
@@ -1317,8 +1322,13 @@ class ProveValEqual : private IterVisitor {
 
   // Inspect a pointwise op and record the identified equality
   void provePwOp(Expr* e) {
-    TensorView* tv = *ir_utils::filterByType<TensorView>(e->outputs()).begin();
-    std::vector<IterDomain*> io = tv->getRootDomain();
+    if (e->output(0)->getValType() != ValType::TensorView) {
+      return;
+    }
+
+    TORCH_INTERNAL_ASSERT(e->outputs().size() == 1);
```
Comment (shmsong) [inline torch/csrc/jit/codegen/cuda/ir_nodes.cpp:1329]: Same as above, isTVOp checks this too.

**Review by shmsong — APPROVED** [review]
Thank you so much for fixing this, and apologies for not covering non-tv op cases in the test in the first place. I just had one minor suggestion, but in general looks good to me.

File: torch/csrc/jit/codegen/cuda/ir_nodes.cpp (hunk 1 above)
Comment (naoyam) [reply torch/csrc/jit/codegen/cuda/ir_nodes.cpp:1226]: Thanks. I thought about that, but `isTVOp` is in lower_utils.h, which I didn't want to include from ir_nodes.cpp. We should move utility functions not tied to lowering, like `isTVOp`, to ir_utils.h.

**Pull request merged**